### DISCUSSION TITLE
embed multiple csh into textual

### DISCUSSION THREAD

**Discussion by etse1288** [Textualize/textual#1512] [Q&A] [answered]
Hi there,
I would like to create an application like `screen` in unix but with a few bell and whistle widget in textual.
I wonder if anyone has done so?
Thanks!

**Comment by davep** [comment, marked as answer]
I'm not aware of anyone having done anything like that with Textual yet, although you may find [this comment](https://github.com/Textualize/textual/issues/1093#issuecomment-1313143926) interesting.
